README.md

```diff
@@ -281,3 +281,33 @@ The test doesn't require any special setup. The instructions to run it are insid
 [utam-js]: https://www.github.com/salesforce/utam-js
 [utam-doc]: https://utam.dev
 [utam-java-recipes]: https://www.github.com/salesforce/utam-java-recipes
+
+## Run Salesforce Mobile test
+
+- Follow the instructions at [Get Started for Mobile](https://utam.dev/guide/get_started_utam#get-started-for-mobile) to set up your local simulator/emulator.
+- Make sure [Appium](https://github.com/appium/appium#readme) and [node.js](https://nodejs.org/en/) are installed on your machine.
+- Update the wdio configuration file:
+For an iOS test, update `wdio.conf.ios.js` file to configure, test device name and os version, the full path for the test application:
+
+```js
+'appium:deviceName': 'iPhone 12',
+'appium:app': '<path to iOS test app>',
+'appium:platformVersion': '15.2',
+```
+
+For an Android test, update `wdio.conf.android.js` file to configure the application bundleid, the full path for the test application and application initial activity:
+
+```js
+'appium:deviceName': 'emulator-5554',
+'appium:app': '<path to Android test app>',
+'appium:appActivity': 'com.salesforce.chatter.Chatter',
+'appium:appPackage': 'com.salesforce.chatter',
+```
+
+- Download the [debug build](https://developer.salesforce.com/tools/mobile-debugging) for SalesforceApp iOS and Android.
+- Commands to execute test: 
+For iOS: yarn test wdio.conf.ios.js
+For Android: yarn test wdio.conf.android.js
+- For a test on Android, make sure to start an emulator before the test run. Otherwise, will hit error like this: "Error: Failed to create session.
```

Comment:
```suggestion
- For a test on Android, make sure to start an emulator before the test run. Otherwise, you will get an error like this: "Error: Failed to create session.
```